.github/workflows/push.yml

```diff
@@ -31,8 +27,6 @@ jobs:
           eval $(docker run --rm -e KBC_DEVELOPERPORTAL_USERNAME -e KBC_DEVELOPERPORTAL_PASSWORD -e KBC_DEVELOPERPORTAL_URL quay.io/keboola/developer-portal-cli-v2:latest ecr:get-login $KBC_DEVELOPERPORTAL_VENDOR $KBC_DEVELOPERPORTAL_APP)
           docker push $REPOSITORY:test
           docker pull quay.io/keboola/syrup-cli:latest
-      - name: Run test job in KBC
```

Comment:
@ondrajodas zmazal som testovaci KBC job. Nedoslo mi, ze to spusta `test` tag na komponente, ... ale procesor sa spusta vzdy najnovsia verzia -> teda nema to ziaden efekt.

Nenasiel sposob, ako spustit `test` tag processora.